docs/events.yaml

```diff
@@ -203,15 +203,15 @@ x-events:
     status: string
     status_info: string
 
-- name: download_image_failed_fetch
+- name: download_image_fetch_failed
   message: "Failed to download image: error fetching from storage backend"
   event_type: infra_env
   severity: "error"
   properties:
     infra_env_id: UUID
 
-- name: host_deregistered_cluster
-  message: "Host {host_name} deregistered"
+- name: host_deregistered
+  message: "Host {host_name} deregistered from cluster"
```

Comment:
didn't we just remove "from cluster" ? 
https://github.com/openshift/assisted-service/commit/cb8576bd58d8e7fa8b45107352e74c0da88aca1b#diff-0aa4b24dc559d5a8af68a084c098a96965b65425c389700c66c4657eb890b38bL214

Reply:
yes, removed